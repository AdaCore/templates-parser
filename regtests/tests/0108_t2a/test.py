from test_support import *

run('templates2ada', ["-e", "tmpl", "-d", ".", "-t", "./localtempl.tads",
                      "-o", "t2a.tmp"] )
file = open("t2a.tmp")
lines = file.readlines()
file.close()

for line in lines:
    print line[:-1].replace("\\", "/")
