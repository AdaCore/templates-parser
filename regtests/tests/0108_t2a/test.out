--  DO NOT EDIT, THIS FILE HAS BEEN GENERATED
package Templates is
   pragma Style_Checks (Off);

   package Lazy is
      Lazy_Block : constant String := "LAZY_BLOCK";
   end Lazy;

end Templates;

--  DO NOT EDIT, THIS FILE HAS BEEN GENERATED
package Templates.Tpl1 is
   pragma Style_Checks (Off);

   Template : constant String := "./tpl1.tmpl";

   This_Is_A_Tag : constant String := "THIS_IS_A_TAG";

end Templates.Tpl1;
--  DO NOT EDIT, THIS FILE HAS BEEN GENERATED
package Templates.Tpl2 is
   pragma Style_Checks (Off);

   Template : constant String := "./tpl2.tmpl";

   N1 : constant String := "N1";
   Name : constant String := "NAME";
   Set_Var : constant String := "SET_VAR";
   Todo : constant String := "TODO";

   package HTTP is
      Hpar : constant String := "HPAR";
      N2 : constant String := "N2";
      Pn : constant String := "PN";
   end HTTP;

   package Set is
      Set_Var : constant String := "azerty";
   end Set;

end Templates.Tpl2;

